docs/topics_5/rn-known-issues.adoc

```diff
@@ -22,11 +22,11 @@ At the time of release, the following known issues are identified as major issue
 
 |link:https://issues.redhat.com/browse/WINDUP-2704[WINDUP-2704]
 |Web UI & Windup-as-a-Service
-|On macOS, the `run_mta.sh` script sets the soft limit of open files to `100000`.
+|On macOS, the `run_mta.sh` script sets the soft limit of open files to `100000`. This issue was resent in MTA version 5.0.0 and resolved in version 5.0.1.
```

Comment:
"resent"? Is this a technical term that I haven't heard? I think "reported" would be clearer. Also, I wouldn't repeat the word "version".